### ISSUE TITLE
Unable to build due to a missing type in `stringifyStringArray.ts`

### ISSUE THREAD

**Issue by marcustut** [brillout/vite-plugin-ssr#161]
I wasn't able to build my project due to an error in the file mentioned, the fix is just to change `str` to `str: string`.

```
yarn run v1.22.15
warning package.json: No license field
$ vite build && vite build --ssr && vite-plugin-ssr prerender
vite v2.6.3 building for production...
✓ 1082 modules transformed.
dist/client/vite-plugin-ssr.json                                    0.07 KiB
dist/client/manifest.json                                           1.29 KiB
dist/client/assets/src/pages/_default.page.client.tsx.32d0876b.js   1.81 KiB / gzip: 0.94 KiB
dist/client/assets/_error.page.19cf351f.js                          0.36 KiB / gzip: 0.22 KiB
dist/client/assets/register.page.33af1e9a.js                        0.89 KiB / gzip: 0.44 KiB
dist/client/assets/Layout.bf6f0a67.js                               4.93 KiB / gzip: 2.31 KiB
dist/client/assets/riddle.page.47c7d71f.js                          1.01 KiB / gzip: 0.54 KiB
dist/client/assets/vendor.c27e8720.js                               469.97 KiB / gzip: 145.38 KiB
node_modules/vite-plugin-ssr/shared/utils/stringifyStringArray.ts:4:32 - error TS7006: Parameter 'str' implicitly has an 'any' type.

4   return '[' + stringList.map((str) => "'" + str + "'").join(', ') + ']'
                                 ~~~


Found 1 error.

error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

**Comment by brillout** [comment]
Released in `0.3.8`.

**Comment by marcustut** [comment]
Yup, I can confirmed it's fixed
